src/gwt/src/org/rstudio/studio/client/common/shell/ShellWidget.java

```diff
@@ -572,6 +581,19 @@ private boolean output(String text,
       if (liveRegion_ != null)
          liveRegion_.announce(output_.getNewText());
 
+      // if we're desktop, write output to console
+      // if (Desktop.isDesktop() && automation_.isEnabled())
+      {
+         if (isError)
+         {
+            Desktop.getFrame().writeStderr(text);
+         }
+         else
+         {
+            Desktop.getFrame().writeStdout(text);
+         }
+      }
```

Comment:
Need to make sure this is only enabled in the main RStudio instance running automation (not the agents, which are spawned by that instance)